Tools/DeployApps/BaasClient.cs

```diff
@@ -158,7 +159,7 @@ private string _shortDifferentiator
             }
         }
 
-        private string _appSuffix => $"-{_shortDifferentiator}-{_clusterName}";
+        private string _appSuffix => $"-{_shortSuffix}";
```

Comment:
If we're running all tests on different clusters, do we even need to have the suffix?

Reply:
We probably still want to differentiate the apps (so that we don't have a gazillion apps with the same name in the UI). If we never look at the UI, then you're right that it doesn't matter.